context/rxjava2/src/test/java/brave/context/rxjava2/CurrentTraceContextAssemblyTrackingTest.java

```diff
@@ -294,6 +295,48 @@ public void parallelFlowable_subscribesUnderScope() {
     subscribesAndRunsInScope2(source, errorSource).assertResult(1, 2, 3);
   }
 
+  /**
+   * This is an example of "micro fusion" where use use a source that supports fusion: {@link
```

Comment:
@akarnokd do you mind reviewing this part? I think this relatively good exercise of the fuseable things I meant to exercise. I really appreciate your blogs. If I got anything conceptually wrong here.. please comment!

Reply:
This is a conditional micro fusion case btw, but the test looks okay.